DataAccess/Infrastructure/OperationDetails.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DataAccess.Infrastructure
+{
+    public class OperationDetails
```

Comment:
you dont need this. If any excapltion happers just try to handle it on in DAL or BL